docs/tutorials/timeseries/forecasting-chronos.ipynb

```diff
@@ -871,30 +1186,54 @@
     "\n",
-    "In several independent evaluations we found Chronos to be effective in zero-shot forecasting. \n",
-    "The accuracy of Chronos-Bolt (base) often exceeds statistical baseline models, and is often comparable to deep learning \n",
-    "models such as `TemporalFusionTransformer` or `PatchTST`.\n",
+    "Alternatively, you can use an ensemble of zero-shot Chronos-2 and fine-tuned Chronos-2 (Small), available via the `chronos2_ensemble` preset:\n",
+    "\n",
+    "```py\n",
+    "predictor = TimeSeriesPredictor(prediction_length=prediction_length).fit(\n",
+    "    ...,\n",
+    "    presets=\"chronos2_ensemble\",\n",
+    ")\n",
+    "```\n",
     "\n",
     "#### What is the recommended hardware for running Chronos models?\n",
     "\n",
-    "For fine-tuning and inference with larger Chronos and Chronos-Bolt models, we tested the AWS `g5.2xlarge` and `p3.2xlarge` instances that feature NVIDIA A10G and V100 GPUs, with at least 16GiB of GPU memory and 32GiB of main memory. \n",
+    "We recommend using a machine with a GPU for best performance, especially for fine-tuning. For reference, we tested the models on AWS `g5.2xlarge` instances with NVIDIA A10G GPUs (24 GiB GPU memory) and 32 GiB of system memory. However, Chronos-2, Chronos-Bolt, and Chronos (up to small size) can also run on consumer GPUs and CPUs with reasonable inference times.\n",
     "\n",
-    "Chronos-Bolt models can also be used on CPU machines, but this will typically result in a longer runtime.\n",
+    "#### Why do my predictions change with the `batch_size`?\n",
     "\n",
+    "By default, AutoGluon enables Chronos-2’s cross_learning mode, where the model makes joint predictions across time series within a batch. This often improves accuracy but also makes results sensitive to the `batch_size`. You can disable this mode with:\n",
+    "\n",
+    "```python\n",
+    "predictor.fit(\n",
+    "    ...,\n",
+    "    hyperparameters={\"Chronos2\": {\"cross_learning\": False}},\n",
+    ")\n",
+    "```\n",
     "\n",
     "#### Where can I ask specific questions on Chronos?\n",
     "\n",
-    "The AutoGluon team are among the core developers of Chronos. So you can ask Chronos-related questions on AutoGluon channels such \n",
+    "Members of the AutoGluon team are among the core developers of Chronos. So you can ask Chronos-related questions on AutoGluon channels such \n",
     "as the Discord [server](https://discord.gg/wjUmjqAc2N), or [GitHub](https://github.com/autogluon/autogluon). You can also join \n",
```

Comment:
Maybe we can remove the Discord server here? We are a lot less active there compared to GitHub